crates/remote/src/config.rs

```diff
@@ -13,6 +13,55 @@ pub struct RemoteServerConfig {
     pub electric_url: String,
     pub electric_secret: Option<SecretString>,
     pub electric_role_password: Option<SecretString>,
+    pub r2: Option<R2Config>,
```

Comment:
Could decouple these into a `ReviewPrConfig` which is optional so that it's possible to build `remote` without specifying review configuration? Could be handy for local dev